includes/content-distribution/class-outgoing-post.php

```diff
@@ -202,15 +202,20 @@ public function get_payload_hash( $payload = null ) {
 	 * @return array|WP_Error The post payload or WP_Error if the post is invalid.
 	 */
 	public function get_payload( $status_on_create = 'draft' ) {
+		$post_author = $this->post->post_author ? Outgoing_Author::get_wp_user_for_distribution( $this->post->post_author ) : [];
+		$multiple_authors = apply_filters( 'newspack_network_multiple_authors_for_post', [], $this->post );
```

Comment:
Can you share more about the use case for this filter? It's missing a docblock.

Reply:
It was an attempt to make it so other implementations of multiple authors (like other than the CAP plugin) could chime in. I'm missing an equivalent on incoming – I'll add that and add those in-function docblocks

Reply:
I also added a filter for incoming "mulitple_authors", so any implementation could hook in in both directions.

Reply:
Ah, I see. I didn't realize that's how the CAP data is being added to the payload.

Non-blocking suggestion: maybe instead of filtering this specific property we can filter the payload `post_data` and let additional features determine their properties. That'll also generate a cleaner payload for sites that don't have CAP at all - no permanent empty array.